File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.java
class Solution {
    public int solution(String binomial) {
        String[] parts = binomial.split(" "); 
        int num1 = Integer.parseInt(parts[0]); 
        String operator = parts[1]; 
        int num2 = Integer.parseInt(parts[2]); 

        if (operator.equals("+")) {
            return num1 + num2;
        } else if (operator.equals("-")) {
            return num1 - num2;
        } else if (operator.equals("*")) {
            return num1 * num2;
        } else {
            return -1; 
        }
    }
}

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.kt
class Solution {
    fun solution(binomial: String): Int {
        var answer = binomial.split(" ")
        return when {
            answer[1] == "+" -> answer[0].toInt() + answer[2].toInt()
            answer[1] == "-" -> answer[0].toInt() - answer[2].toInt()
            answer[1] == "*" -> answer[0].toInt() * answer[2].toInt()
            else -> -1
        }

    }
}